src/bench/bench.cpp

```diff
@@ -55,21 +55,20 @@ bool benchmark::State::KeepRunning()
     else {
         now = gettimedouble();
         double elapsed = now - lastTime;
-        double elapsedOne = elapsed * countMaskInv;
+        double elapsedOne = elapsed / (countMask + 1);
```

Comment:
In commit "Remove countMaskInv caching in bench framework".

Isn't countMask always a power of two minus one? If so, we could just remember the number of bits in it, and use a shift instead of a division?

Reply:
I'm gonna leave this be, I think we need to re-write the bench crap anyway (there is a very massive difference between runs based on the number of iterations, in many of my tests, so all of this garbage needs to just be removed).